keps/sig-node/5055-device-taints-and-tolerations/README.md

```diff
@@ -0,0 +1,971 @@
+- [ ] (R) Graduation criteria is in place
+  - [ ] (R) [all GA Endpoints](https://github.com/kubernetes/community/pull/1806) must be hit by [Conformance Tests](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/conformance-tests.md) 
+- [ ] (R) Production readiness review completed
+- [ ] (R) Production readiness review approved
+- [ ] "Implementation History" section is up-to-date for milestone
+- [ ] User-facing documentation has been created in [kubernetes/website], for publication to [kubernetes.io]
+- [ ] Supporting documentation—e.g., additional design documents, links to mailing list discussions/SIG meetings, relevant PRs/issues, release notes
+
+<!--
+**Note:** This checklist is iterative and should be reviewed and updated every time this enhancement is being considered for a milestone.
+-->
+
+[kubernetes.io]: https://kubernetes.io/
+[kubernetes/enhancements]: https://git.k8s.io/enhancements
+[kubernetes/kubernetes]: https://git.k8s.io/kubernetes
+[kubernetes/website]: https://git.k8s.io/website
+
+## Summary
+
+<!--
+This section is incredibly important for producing high-quality, user-focused
+documentation such as release notes or a development roadmap. It should be
+possible to collect this information before implementation begins, in order to
+avoid requiring implementors to split their attention between writing release
+notes and implementing the feature itself. KEP editors and SIG Docs
+should help to ensure that the tone and content of the `Summary` section is
+useful for a wide audience.
+
+A good summary is probably at least a paragraph in length.
+
+Both in this section and below, follow the guidelines of the [documentation
+style guide]. In particular, wrap lines to a reasonable length, to make it
+easier for reviewers to cite specific portions, and to minimize diff churn on
+updates.
+
+[documentation style guide]: https://github.com/kubernetes/community/blob/master/contributors/guide/style-guide.md
+-->
+
+With Dynamic Resource Allocation (DRA), DRA drivers publish information about
+the devices that they manage in ResourceSlices. This information is used by the
+scheduler when selecting devices for user requests in ResourceClaims.
+
+With this KEP, DRA drivers can mark devices as tainted such that they won't be
+used for scheduling new pods. In addition, pods already running with access to
+a tainted device can be stopped automatically. Cluster administrators can do
+the same by creating a
+[ResourceSliceOverride](../5027-dra-admin-controlled-device-attributes] with a
+taint.
+
+Users can decide to ignore specific taints by adding tolerations to their
+ResourceClaim.
+
+## Motivation
+
+
+### Goals
+
+- Enable taking devices offline for maintenance while still allowing tests pods
+  to request and use those devices.
+
+- Enable users to decide whether they want to keep running a workload in a degraded
+  mode while a device is unhealthy or prefer to get pods rescheduled.
```

Comment:
Running specific workloads on specific devices with taints & tolarations, is probably obvious enough goal that it does not need to be stated here.   But maybe few extra details will raise additional thoughts of the possibilities:
```suggestion
- Enable taking devices offline for maintenance while still allowing tests pods
  to request and use those devices.  Being able to do this device at a time, minizes
  service level disruption.

- Enable users to decide whether they want to keep running separate workloads, or
 a subset of existing workloads, in a degraded  mode while a device is unhealthy, or
 prefer to get pods rescheduled.
```

Reply:
I updated the first point. But the changes to the second one seem unnecessarily complex, so I kept it as-is.